tests/integration/src/callbackauthn/b_saml.py

```diff
@@ -31,7 +31,7 @@ def test_create_auth_domain(
     get_token: Callable[[str, str], str],
 ) -> None:
     # Create a saml client in the idp.
-    create_saml_client("saml.integration.test", "/api/v1/complete/saml")
+    create_saml_client(f"{signoz.self.host_configs['8080'].address}:{signoz.self.host_configs['8080'].port}", "/api/v1/complete/saml")
```

Comment:
```suggestion
    create_saml_client(f"{signoz.self.host_configs['8080'].address}:{signoz.self.host_configs['8080'].port}", "/api/v1/complete/saml")
```

Do we need this change?